### PR TITLE
Document annotation endpoints.kubernetes.io/last-change-trigger-time

### PR DESCRIPTION
**Pull request by kundan2707**
Fixes: #3800

### CODE REVIEW

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -1086,6 +1086,18 @@ has been truncated to 1000.
...
 
+### endpoints.kubernetes.io/last-change-trigger-time
+
+Type: Annotation
+
+Example: `endpoints.kubernetes.io/last-change-trigger-time: "2022-01-25T20:39:38Z"`
+
+Used on: Endpoints
+
+This annotation set to an [Endpoints](/docs/concepts/services-networking/service/#endpoints) object
```
Comment (windsonsea) [inline content/en/docs/reference/labels-annotations-taints/_index.md:1097]:
```suggestion
This annotation is set on an [Endpoints](/docs/concepts/services-networking/service/#endpoints) object
```

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -1086,6 +1086,18 @@ has been truncated to 1000.
...
+that represents the timestamp (stored as RFC 3339 date-time string, e.g. '2018-10-22T19:32:52.1Z') of the last change.
+This timestamp belongs to some Pod or Service object, that triggered the endpoints object change.
```
Comment (windsonsea) [inline content/en/docs/reference/labels-annotations-taints/_index.md:1099]:
```suggestion
This timestamp belongs to either a Pod or Service object, that triggered the change in the endpoints object.
```

**Review by windsonsea — COMMENTED** [review]
nits

**Comment by kundan2707** [comment]
@windsonsea  incorporated changes suggested by you

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 3]
```diff
@@ -1086,6 +1086,18 @@ has been truncated to 1000.
...
+
+Type: Annotation
+
+Example: `endpoints.kubernetes.io/last-change-trigger-time: "2022-01-25T20:39:38Z"`
+
+Used on: Endpoints
+
+This annotation is set on an [Endpoints](/docs/concepts/services-networking/service/#endpoints) object
+that represents the timestamp (stored as RFC 3339 date-time string, e.g. '2018-10-22T19:32:52.1Z') of the last change.
+This timestamp belongs to either a Pod or Service object, that triggered the change in the endpoints object.
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:1099]:
```suggestion
This timestamp belongs to either a Pod or Service object, that triggered the change in the Endpoints object.
```
How does ”belong” fit into this sentence? I don't understand what it means.

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 4]
```diff
@@ -1086,6 +1086,18 @@ has been truncated to 1000.
...
+### endpoints.kubernetes.io/last-change-trigger-time
+
+Type: Annotation
+
+Example: `endpoints.kubernetes.io/last-change-trigger-time: "2022-01-25T20:39:38Z"`
+
+Used on: Endpoints
+
+This annotation is set on an [Endpoints](/docs/concepts/services-networking/service/#endpoints) object
+that represents the timestamp (stored as RFC 3339 date-time string, e.g. '2018-10-22T19:32:52.1Z') of the last change.
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:1098]:
```suggestion
that represents the timestamp (stored as RFC 3339 date-time string, such as `2018-10-22T19:32:52.1Z`)
of the most recent change to the set of endpoints.
```

**Comment by kundan2707** [comment]
@sftim  incorporated review comments

**Comment by sftim** [comment]
/retitle Document annotation endpoints.kubernetes.io/last-change-trigger-time

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 3 above)
Comment (sftim) [reply content/en/docs/reference/labels-annotations-taints/_index.md]: How does ”belong” fit into this sentence? I don't understand what it means.

**Comment by sftim** [comment]
I have a [question](https://github.com/kubernetes/website/pull/42245#discussion_r1280474186) still.

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 3 above)
Comment (kundan2707) [reply content/en/docs/reference/labels-annotations-taints/_index.md]:
```
This annotation is set on an [Endpoints](/docs/concepts/services-networking/service/#endpoints) object
that represents the timestamp (stored as RFC 3339 date-time string, e.g. '2018-10-22T19:32:52.1Z') of the last change, of some Pod or Service object, that triggered the endpoints object change.
```
This one will be better

Comment (sftim) [reply content/en/docs/reference/labels-annotations-taints/_index.md]: > This timestamp belongs to either a Pod or Service object, that triggered the change in the Endpoints object.

I still don't quite follow.

**Comment by sftim** [comment]
Even without the wording being exactly right, I'd like to get a tech review.

/sig network

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 3 above)
Comment (tengqm) [reply content/en/docs/reference/labels-annotations-taints/_index.md]: Agreed. This "belongs to" is confusing. Better revise.
/approve cancel

**Comment by sftim** [comment]
@kundan2707 I'm afraid you might need to rebase - if that's new to you, check out https://www.youtube.com/watch?v=gkGZzd9c4ow, https://www.youtube.com/watch?v=Dv8I_kfrFWw and https://www.youtube.com/watch?v=1ffBJ4sVUb4

**Comment by divya-mohan0209** [comment]
@kundan2707 : Please could you rebase correctly? It seems there are way too many unexpected commits on here.

**Comment by kundan2707** [comment]
@divya-mohan0209  yes it seems branch got rebased incorrectly as there were so much time and changes  in between. I will correct it.

**Pull request closed without merging**